initial goerli branch

brossetti1 opened the pull request:
## GitHub Issue

https://github.com/HausDAO/daohaus-app/issues/1911

## Changes

Goerli block explorer: https://goerli.etherscan.io/


I have branched off all the open prs as follows:
```
master <---- deploy-goerli <----- remove-rinkeby-koven
```

#### other WIP branches:

- [update HausDAO boosts](https://github.com/HausDAO/daohaus-boosts/pull/4)
- [update HausDAO stats](https://github.com/HausDAO/daohaus-stats/pull/24)
- [update HausDAO wrapnzap](https://github.com/HausDAO/wrap-n-zap/pull/5) 
- [update HausDAO Supergraph](https://github.com/HausDAO/daohaus-supergraph/pull/68)
- [update HausDAO website docs](https://github.com/HausDAO/daohaus-website/pull/187)
- [update HausDAO MinionSummonerV2](https://github.com/HausDAO/MinionSummonerV2/pull/25)
- [update Moloch2.1](https://github.com/HausDAO/Molochv2.1/pull/12)
- [update Moloch2.5](https://github.com/HausDAO/Moloch2.5/pull/15)
- [update erc721 subgraph](https://thegraph.com/hosted-service/subgraph/odyssy-automaton/erc721-goerli-subgraph)
- [update erc1155 subgraph](https://thegraph.com/hosted-service/subgraph/odyssy-automaton/erc1155-goerli-subgraph)

# Chain Details

```
{
    moloch_factory_addr: '0x72B8Bf40C8B316753a3E470689DA625759D2b025',
    wrapper_contract: '0xB4FBF271143F4FBf7B91A5ded31805e42b2208d6',
    wrap_n_zap_factory_addr: '0xf89f79A0E5aF89BFa5c4d4FC6F7fD25700bC4905',
    escrow_minion: '0xceE7f251Bd38B21E8F71C1d62cFbb18219a7F606',
    block_explorer: 'https://goerli.etherscan.io/',
    safeMinion: {
      minion_factory_addr: '0x121931c0Bc458A5f13F3861444AeB036cc8a5363',
      safe_mutisend_addr: '0xA238CBeb142c10Ef7Ad8442C6D1f9E89e07e7761',
      safe_sign_lib_addr: '0xa25b3579a295be016de5eb5F082b54B12d45F72C',
    },
    disperse_app: '0x3D0e848b6C55153E2b0154734ac6b5288A7f1B6F',
    poster: '0x3c1f4802be7e26d95b31ef7a566e18f42e360cab',
}
```

# Additional Resources

https://github.com/goerli/testnet - rpc urls, faucets, etc
working faucet - https://goerli-faucet.pk910.de/ - POW based


## Packages Added

Brief list of any packages added, and if folks need to rerun `yarn install` to run locally

## Checks

Before making your PR, please check the following:

- [ ] Critical lint errors are resolved
- [ ] App runs and builds locally

brossetti1 committed:
adding goerli (+7 squashed commits)
Squashed commits:
[737b94a3] updating config
[4c5dfa55] configuring goerli
[2683aed5] adding goerli to the rest of daohaus
[8409f1f4] typo fix
[4322e37a] adding goerli config
[0aa23d5f] adding config for contract deploys
[c47d6715] initial goerli branch

skuhlmann commented on src/utils/chain.js:
```diff
@@ -167,6 +167,66 @@ export const supportedChains = {
+    // 0xA8680d0E43aDe8BC32eCCF6C446eCe4CA15d4258 - molochsummmoner
+    // 0x0080C8Ec84E1CA7E8A2c68FcCFB6C42f4D2103bD
+    // moloch_factory_addr: '0x38064F40B20347d58b326E767791A6f79cdEddCe',
+    minion_factory_addr: '0x25D59ad2b29e25221b844Ae3F406Ff9B3bC90f14',
```
v1 minion - we don't need this

skuhlmann commented on src/utils/chain.js:
```diff
@@ -167,6 +167,66 @@ export const supportedChains = {
+    // moloch_factory_addr: '0x38064F40B20347d58b326E767791A6f79cdEddCe',
+    minion_factory_addr: '0x25D59ad2b29e25221b844Ae3F406Ff9B3bC90f14',
+    moloch_factory_addr: '0x72B8Bf40C8B316753a3E470689DA625759D2b025',
+    dai_contract: '0x11fE4B6AE13d2a6055C8D9cF65c55bac32B5d844',
```
i don't think we need this.

skuhlmann commented on src/utils/chain.js:
```diff
@@ -167,6 +167,66 @@ export const supportedChains = {
+    moloch_factory_addr: '0x72B8Bf40C8B316753a3E470689DA625759D2b025',
+    dai_contract: '0x11fE4B6AE13d2a6055C8D9cF65c55bac32B5d844',
+    wrapper_contract: '0xB4FBF271143F4FBf7B91A5ded31805e42b2208d6',
+    wrap_n_zap_factory_addr: '0xf89f79A0E5aF89BFa5c4d4FC6F7fD25700bC4905',
```
nice to have but not essential

skuhlmann commented on src/utils/chain.js:
```diff
@@ -167,6 +167,66 @@ export const supportedChains = {
+      safe_sign_lib_addr: '0xa25b3579a295be016de5eb5F082b54B12d45F72C',
+    },
+    // deprecated
+    superfluid: {
```
don't need any of the superfluid stuff

skuhlmann commented on src/utils/chain.js:
```diff
@@ -167,6 +167,66 @@ export const supportedChains = {
+      supertoken_factory: '0x94f26B4c8AD12B18c12f38E878618f7664bdcCE2',
+      version: 'v1',
+    },
+    disperse_app: '0x3D0e848b6C55153E2b0154734ac6b5288A7f1B6F',
```
only if there is one for goerli

brossetti1 replied:
i deployed this one - figured it doesnt hurt to have it on goerli

brossetti1 committed:
adding etherscan (+5 squashed commits)
Squashed commits:
[999617b1] adding safes
[a2178951] fixxes summon easy mode
[047320db] adding shaman subgraph
[d664623e] update subgraphs
[2e8d5517] adding correct subgraph endpoints for goerli
